panda/src/downloader/bioPtr.cxx

```diff
@@ -95,6 +95,17 @@ BioPtr(const URLSpec &url) : _connecting(false) {
     // These hints tell getaddrinfo what kind of address to return.
     struct addrinfo hints, *res = nullptr;
     memset(&hints, 0, sizeof(hints));
+
+// For some reason in msys those two macros are not defined correctly in the
+// header file ws2tcpip.h
```

Comment:
Please update comment to imply that these should be removed when _WINNT_VER (or whatever it was called) is changed, so we don't forget about it.

Reply:
Could also encapsulate this with `#if _WIN32_WINNT < 0x0600` similar to here: https://github.com/panda3d/panda3d/blob/95f3e30d376a8c3c744cc20d68cbe8b8254db187/dtool/src/dtoolbase/mutexWin32Impl.cxx#L24

Reply:
I don't think it's necessary because the `#ifndef` already makes sure those definitions won't be redefined, also, with other compilers the definitions works fine regardless of the `_WIN32_WINNT` macro and these definitions would be redefined again.